test/recipes/30-test_evp_data/evppkey_ffdhe.txt

```diff
@@ -93,6 +93,22 @@ PeerKey=ffdhe2048-2-pub
 Ctrl = dh_pad:1
 SharedSecret=00006620DD85B56EE8540C8040CAC46B7385344A164E4DBDF521F7D99F88FA68EDD295A45E36E0BBD5FF5DE84598824E2CA52ED82ACA918CAECC6B22846D0FC6F0203E8B6963964D11E9E704F83AF1D60E9B1931139E9E9967C4665A831A75D99359A8BA80DD5921E74379AF4CA8DB453EDBC5E669AB17A5254CA6C96794CD5196BE90AF37742C8F6812515FFCC45B08F4158EFF9559F1AEF3665B3D91519DCBC6DF22CD6DA521B86613558602E73D2CA4666972F7D2CB6B46299B1DF2DA29A2A2D99D105E10CB553D6738A9B1DB2A0314C3CF30642D5C44695623D8B95C4426BEA830FB51816B4F086945E9B12A445F42DD68610E3F378A6E69A383D13D85BF
 
+Derive=ffdhe2048-1
```

Comment:
Maybe a comment for what this is testing? Are these just arbitrary values used here?

Reply:
I agree, there should be a comment here.

Do we know that one of these cases hit the leading zero problem?

Reply:
> Maybe a comment for what this is testing? Are these just arbitrary values used here?

I'll add a comment. Those keys are commented above that they create a shared secret with leading zeros.

Also I've tested that with the fix reverted this testcase fails.